internal/store/all_clusters_projects.go

```diff
@@ -0,0 +1,24 @@
+package store
+
+import (
+	"context"
+	"fmt"
+
+	om "github.com/mongodb/go-client-mongodb-ops-manager/opsmngr"
+	"github.com/mongodb/mongocli/internal/config"
+)
+
+type ListAllClusters interface {
```

Comment:
it's a go convention to name single method interfaces as `er` adjectives, this should be `AllClustersLister`